docs/source/howto/query.rst

```diff
@@ -0,0 +1,415 @@
+.. _how-to:query:
+
+*******************************
+How to find and query for data
+*******************************
```

Comment:
I did not review this section in detail assuming that it is largely similar to the original section in "exploring". But please let me know if this is not the case and I should read this more thoroughly.